model-setup-examples/basic-elevon.md

```diff
@@ -0,0 +1,36 @@
+## Basic Elevon
+
+Elevons are control surfaces that combine the functions of the elevator and the aileron.
+
+You usually see this sort of configuration on tailless aircraft.
+
+A good example of this is an aircraft known asa "flying wing".
+
+![](/assets/flying-wing.jpg)
+
+As you can see from the picture above, the airleron and elevator surfaces reside in the same physical control surface.
+
+For this to work, you have to create a "mix" because this plane has only one servo connected to the Aileron aileron port of the reciever \(Channel 1\), and one servo connected to the Elevator port of the receiver \(Channel 2\).  
+
```

Comment:
Didn't think it was relevant to talk about the third channel at all since it has nothing to do with elevon mixing, however, probably wouldn't hurt anything if I pointed it out.